Marlin/stepper.cpp

```diff
@@ -459,6 +474,10 @@ void Stepper::isr() {
       // #if ENABLED(ADVANCE)
       //   e_steps[TOOL_E_INDEX] = 0;
       // #endif
+
+      #if ENABLED(MK2_MULTIPLEXER)
+        select_multiplexed_stepper(current_block->active_extruder);
```

Comment:
> It could even be possible to drive multiple steppers with the multiplexer "simultaneously" by switching the multiplexer and sending step pulses to each stepper

Let's walk before we fly. The multiplexer needs to be implemented faithfully as it works in the Prusa fork.